src/Raven.Client/Documents/Queries/TimeSeries/TimeSeriesAggregation.cs

```diff
@@ -145,7 +145,7 @@ public class TimeSeriesAggregationResult<T> : TimeSeriesQueryStreamResultBase<Ti
     public class TimeSeriesRangeAggregation : IPostJsonDeserialization, ITimeSeriesQueryStreamEntry
     {
         public long[] Count;
-        public double[] Max, Min, Last, First, Average, Sum;
+        public double[] Max, Min, Last, First, Average, Sum, Percentile, Slope, StandardDeviation;
```

Comment:
Should we maybe shorten this to StdDev?

Reply:
yes

Reply:
No, then Average should be Avg, etc. Either use full names of abbreviations. Not both.

Reply:
Actually it is already mixed, nvm.

Reply:
May I ask what's definition used for slope?

Reply:
the rate of change within a range (only works with group by)
i.e. `slope = dY/dX`, dY = `range.LastVal - range.FirstVal`, dX = `(range.End - range.Start).TotalMilliseconds`

Reply:
ok I thought  it uses linear regression